zerver/webhooks/gitlab/view.py

```diff
@@ -124,15 +124,29 @@ def get_merge_request_open_or_updated_body(payload: Dict[str, Any], action: str,
         pull_request.get('source_branch'),
         pull_request.get('target_branch'),
         pull_request.get('description'),
-        get_objects_assignee(payload),
+        assignees=get_objects_assignees(payload),
         type='MR',
         title=payload['object_attributes'].get('title') if include_title else None
     )
 
-def get_objects_assignee(payload: Dict[str, Any]) -> Optional[str]:
+def get_objects_assignees(payload: Dict[str, Any]) -> List[Dict[str, Any]]:
+    assignees_data = []
     assignee_object = payload.get('assignee')
     if assignee_object:
-        return assignee_object.get('name')
+        assignees_object = payload.get('assignee')
```

Comment:
Are you making sure that the old payloads work too? I am not sure we need to account for outdated payloads, but we can chat about that at some point for sure!

Reply:
This allows the merge requests to stay working, as the fixture for them uses assignee rather than assignees.

Reply:
Ok cool! Sounds good! Thanks for clarifying! :)